scripts/fallback-template-long.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
Prefer to use /usr/bin/env to reference the user's bash (sometimes it's not in /usr/bin):
```suggestion
#!/usr/bin/env bash
```